### PR TITLE
Conda env

### LINKED ISSUE #17: Create conda dependencies file with sphinx
Once we start changing the pandas contribution documentation (#12) we'll have to generate the html version of it. This is being done in most Python projects with [Sphinx](http://www.sphinx-doc.org/en/master/). To make it easier to create a development environment for this, we can use conda, and specify the dependencies in a file named `environment.yml`. We can base it on the pandas one, but for now we'll just add sphinx (the same version as pandas).

### PR DESCRIPTION
**Pull request by dujm**
add conda-env sphinx, Closes #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-# pandas-mentoring
-<img src="img/logo.jpeg" width="600" height="200"><br>
+# Mentoring new pandas contributors (team 2019)
+
+This is the list of the [GitHub](https://github.com) usernames of all the people interested in contributing to [pandas](https://github.com/pandas-dev/pandas):
```
Comment (datapythonista) [inline README.md:3]: I guess this was from previous work, seems unrelated to this PR.

Happy to get proposals for the README, but if you want to do, better in a separate PR.

Comment (dujm) [reply README.md:3]: @datapythonista sorry and yes, the ```README``` change was from previous work (I should not have added it in my PR). Now I submitted a new PR (#40) which only contains the change in the ```environment.yml```
